docs/UserGuide.md

```diff
@@ -110,12 +110,12 @@ The parameters are :
 * `w/WEIGHT` is the weight of patient in kilograms rounded to the nearest integer.
 * `b/BLOOD_TYPE` is chosen from one of the 4 choices, `A/B/AB/O`.
 * `p/PHONE_NUMBER` must be 8-digit Singapore phone number.
-* `o/MEDICAL_CONDITION` is the list of patient's past/pre-existing medical conditions. For medical condition that has multiple
+* `m/MEDICAL_CONDITION` is the list of patient's past/pre-existing medical conditions. For medical condition that has multiple
```

Comment:
Have you tried to run the tests, and how this would change with `add t/patient m/sth sth` or tried to run in AB3 itself